core/basepage.php

```diff
@@ -257,8 +255,25 @@ public function display(): void
         } else {
             print "Error: Headers have already been sent to the client.";
         }
+    }
+
+    public function flush_output(): void
+    {
+
+    }
```

Comment:
This seems an unused stub? I guess it was added here first, then moved to polyfills.php, but not removed from here?

Reply:
Yeah, I think that's what happened. I' removed it from my code, i'll commit it to this request.